exchanges/refocus/src/refocusExchange.ts

```diff
@@ -0,0 +1,50 @@
+import { pipe, tap } from 'wonka';
+import { Exchange, Operation } from '@urql/core';
+
+export const refocusExchange = (): Exchange => {
+  return ({ client, forward }) => ops$ => {
+    const watchedOperations = new Map<number, Operation>();
+    const observedOperations = new Map<number, number>();
+    const keys: Array<number> = [];
+
+    window.addEventListener('focus', () => {
+      keys.forEach(key => {
```

Comment:
If we're already using `forEach` here we could also call `watchedOperations.forEach`?